jobsdb/backup.go

```diff
@@ -137,15 +138,30 @@ func (jd *Handle) uploadDumps(ctx context.Context, dumps map[string]string) erro
 		path := filePath
 		operation := func() error {
 			if err := jd.uploadTableDump(ctx, wrkId, path); err != nil {
-				jd.logger.Errorf("[JobsDB] :: Failed to upload workspaceId %v. Error: %s", wrkId, err.Error())
-				stats.Default.NewTaggedStat("backup_ds_failed", stats.CountType, stats.Tags{"customVal": jd.tablePrefix, "workspaceId": wrkId}).Increment()
+				jd.logger.Errorw(
+					"failed backup upload",
+					constants.WorkspaceID, wrkId,
+					constants.Err, err.Error(),
+				)
+				stats.Default.NewTaggedStat(
+					"backup_ds_failed",
+					stats.CountType,
+					stats.Tags{
+						constants.CustomVal: jd.tablePrefix,
+						"workspaceId":       wrkId,
```

Comment:
this is an example why the same constants for logs and stats might not fit the bill

Reply:
using the same everywhere would simplify our lives, though.
I'll change this to use `workspaceID` too(shouldn't cause problems because we don't use this stat as much now), the others can follow soon.

Reply:
Isn't `workspaceId` an important stat label though?
My personal opinion is that we should:
1. Not mix log fields and stat tags
2. Use 1 common set of log field constants for rudder-server (warehouse can use a separate one)
3. Use 1 common set of stat tag constants

Reply:
why not mix them..? after all they're only tags - either stats or logs.
I don't see the kind of problems it could cause, please elaborate

Reply:
yes
I was expressing the need to have the same in both places